scripts/polaris/tiff_to_zarr.py

```diff
@@ -58,8 +71,8 @@ def main():
     backend = detect_cli_io_backend(file_paths)
     image = cli_input_to_ngff_image(backend, file_paths)
     # The scale and axis units are the same as the one printed in the reconstruction script
-    image.scale = {'z': 0.65, 'y': 0.65, 'x': 0.65}
-    image.axes_units = {'z': 'micrometer', 'y': 'micrometer', 'x': 'micrometer'}
+    image.scale = read_pixelsize_from_hdf5(args.raw_directory)
+    image.axes_units = {'x': 'micrometer', 'y': 'micrometer', 'z': 'micrometer'}
```

Comment:
I think you can get units from the h5 file as well.

Reply:
The pixel_size attribute seems to only store the numerical value but not the units. The name of the camera model is also listed in the h5 file (PCO_Edge), but not the units. The units are defined in a comment in one of Dula's Jupyter notebooks (`notebooks/microct/backend/ALS_recon_functions.py`)

Here are the attributes stored in `/measurement/instrument/detector/` in the h5 file:
```{'binning_x': array([1.]),
'binning_y': array([1.]),
'dark_field_value': array([100.]),
'delay_time': array([0.]),
'dimension_x': array([2560.]),
'dimension_y': array([2160.]),
'exposure_time': array([0.15, 0.15, 0.15, ..., 0.15, 0.15, 0.15]),
'model': b'PCO_Edge',
'pixel_size': array([0.00065]),
'temperature': array([25.])}```